src/DesktopTests/UIAutomation/CustomObjects/RegistrarTests.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 using System;
+using System.IO;
```

Comment:
Is `using System.IO` required?

Reply:
Yes, unless you'd prefer I replace `InvalidDataException` with `System.io.InvalidDataException`.